clients/src/main/java/org/apache/kafka/common/requests/ListOffsetsRequest.java

```diff
@@ -49,6 +49,8 @@ public class ListOffsetsRequest extends AbstractRequest {
 
     public static final long LATEST_TIERED_TIMESTAMP = -5L;
 
+    public static final long EARLIEST_PENDING_UPLOAD_OFFSET_TIMESTAMP = -6L;
```

Comment:
consider renaming it to `EARLIEST_PENDING_UPLOAD_TIMESTAMP` for uniformity with other names.

Reply:
Done